app/models/user.rb

```diff
@@ -45,7 +47,12 @@ def confirmed?
     !confirmed_at.blank?
   end
 
-  def send_confirmation
+  def confirm
+    update_attributes(confirmation_token: nil, confirmed_at: Time.now.utc)
+  end
+
+  def send_confirmation(unconfirm = false)
+    self.confirmed_at = nil if unconfirm
```

Comment:
I updated this method to avoid duplicated code.. we need this method, and the one above for active admin.